src/components/human-impact-layers/human-impact-layers-component.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { loadModules } from '@esri/react-arcgis';
+
+import MultipleActiveLayers from 'components/multiple-active-layers';
+
+import { HUMAN_PRESSURE_LAYER_ID } from 'constants/human-pressures';
+
+const HumanImpactLayers = ({ map, title, rasters, setRasters, description, options, setLayerVisibility, theme, activeLayers }) => {
+  const alreadyChecked = options.reduce((acc, option) => ({ 
+    ...acc, [option.value]: rasters[option.value]
+  }), {});
+
+  const handleHumanPressureRasters = (rasters, option) => {
+    const { layers } = map;
+    const humanImpactLayer = layers.items.find(l => l.id === HUMAN_PRESSURE_LAYER_ID);
+    setRasters(rasters);
+
+    if (Object.values(rasters).some(raster => raster)) {
+      setLayerVisibility(HUMAN_PRESSURE_LAYER_ID, true);
+    } else {
+      setLayerVisibility(HUMAN_PRESSURE_LAYER_ID, false);
+    }
```

Comment:
This syntax can be a bit shorter, maybe you can find better name for `hasRastersWithData`:

`const hasRastersWithData = Object.values(rasters).some(raster => raster);`
`setLayerVisibility(HUMAN_PRESSURE_LAYER_ID, hasRastersWithData);`

Reply:
thanks for that catch